### ISSUE TITLE
Use global parameters list from discovery to check method parameters.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#127]
```
Use global parameters list from discovery to check method parameters.

https://developers.google.com/discovery/v1/reference#resource_discovery

```

Original issue reported on code.google.com by `jcgregorio@google.com` on 19 Apr 2012 at 3:00

**Comment by GoogleCodeExporter** [comment]
```
Out for review http://codereview.appspot.com/6299050/
```

Original comment by `jcgregorio@google.com` on 6 Jun 2012 at 4:19

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 98d715ff68bf.
```

Original comment by `jcgregorio@google.com` on 6 Jun 2012 at 6:07
- Changed state: **Fixed**
